bamt/networks/base.py

```diff
@@ -91,7 +93,7 @@ def add_nodes(self, descriptor: Dict[str, Dict[str, str]]):
         descriptor: dict with types and signs of nodes
         """
         if not self.validate(descriptor=descriptor):
-            if not self.type == "Hybrid":
+            if not self.type == "Hybrid" or not self.type == "Composite":
```

Comment:
Has CompositeBN got `self.validate` method? Can't find it.. It seem, all method of learning were overwritten

Reply:
I've added it just in case, for the future